LICENSE.md

```diff
@@ -0,0 +1 @@
+Add one
```

Comment:
There's already a file called `LICENSE`. You could just use that instead of this one. 😃